src/app/zap-templates/zcl/data-model/chip/keypad-input-cluster.xml

```diff
@@ -26,26 +26,26 @@ limitations under the License.
     <!-- TODO: Add feature map once it is available -->
     <description>This cluster provides an interface for controlling a device like a TV using action commands such as UP, DOWN, and SELECT.</description>
 
-    <command source="client" code="0x00" name="SendKey" optional="false">
+    <command source="client" code="0x00" name="SendKeyRequest" response="SendKeyResponse" optional="false">
       <description>Upon receipt, this SHALL process a keycode as input to the media device.</description>
-      <arg name="keyCode" type="KeypadInputCecKeyCode"/>
+      <arg name="keyCode" type="CecKeyCode"/>
     </command>
 
-    <command source="server" code="0x00" name="SendKeyResponse" optional="false">
+    <command source="server" code="0x01" name="SendKeyResponse" optional="false">
       <description>This command SHALL be generated in response to a SendKey Request command.</description>
-      <arg name="status" type="KeypadInputStatus"/>
+      <arg name="statusEnum" type="StatusEnum"/>
     </command>
 
   </cluster>
 
-  <enum name="KeypadInputStatus" type="ENUM8">
```

Comment:
Please remove from list in src/app/zap-templates/templates/app/helper.js

Reply:
Removed.